server/controllers/user.js

```diff
@@ -130,4 +131,30 @@ module.exports = {
       res.status(500).send("Internal Server Error");
     }
   },
+
+  // Complete and close review by checking amount of messages and applying rating and status changes
+  async closeReview(req, res) {
+    try {
+      const { reviewId, rating, complete } = req.body;
+
+      const review = await Review.findOne({ reviewId });
+
+      if (review.messages.length > 2 && complete) {
+        await Review.findByIdAndUpdate(
+          { _id: reviewId },
+          { $set: { rating: rating, status: "closed" } }
```

Comment:
it is good using $set here, but may not need it since it is a findByIdAndUpdate

Reply:
Fair, I was just following what I've seen used. I can correct this.